pkg/deployment/deployment.go

```diff
@@ -607,3 +607,17 @@ func (builder *Builder) WithToleration(toleration coreV1.Toleration) *Builder {
 
 	return builder
 }
+
+// WithSchedulerName configures a scheduler to process pod's scheduling.
+func (builder *Builder) WithSchedulerName(schedulerName string) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Setting scheduler %v for deployment %s in namespace %s",
```

Comment:
schedulerName is sting var, please change formatter from %v to %s

Reply:
Please change log message to:
>Adding scheduler %s to deployment %s in namespace %s

Reply:
Updated to use `%s`